buildengine/engine_test.go

```diff
@@ -16,7 +16,7 @@ func TestEngine(t *testing.T) {
 		t.SkipNow()
 	}
 	ctx := log.ContextWithNewDefaultLogger(context.Background())
-	engine, err := buildengine.New(ctx, nil, []string{"testdata/alpha", "testdata/other", "testdata/another"})
+	engine, err := buildengine.New(ctx, nil, t.TempDir(), []string{"testdata/alpha", "testdata/other", "testdata/another"})
```

Comment:
Just a heads up that `t.TempDir()` will create a new tempdir each time it's called. Just make sure that's what you want.

Reply:
I had to look that up while I was working on this :)